ui/components/multichain/network-list-menu/add-non-evm-account/add-non-evm-account.tsx

```diff
@@ -39,10 +40,22 @@ const SNAP_CLIENT_CONFIG_MAP: Record<
     clientType: WalletClientType.Bitcoin,
     chainId: MultichainNetworks.BITCOIN_TESTNET,
   },
+  [MultichainNetworks.BITCOIN_SIGNET]: {
+    clientType: WalletClientType.Bitcoin,
+    chainId: MultichainNetworks.BITCOIN_SIGNET,
+  },
   [MultichainNetworks.SOLANA]: {
     clientType: WalletClientType.Solana,
     chainId: MultichainNetworks.SOLANA,
   },
+  [MultichainNetworks.SOLANA_TESTNET]: {
+    clientType: WalletClientType.Solana,
+    chainId: MultichainNetworks.SOLANA,
+  },
+  [MultichainNetworks.SOLANA_DEVNET]: {
+    clientType: WalletClientType.Solana,
+    chainId: MultichainNetworks.SOLANA,
```

Comment:
```suggestion
    chainId: MultichainNetworks.SOLANA_DEVNET,
```

Reply:
Done in https://github.com/MetaMask/metamask-extension/pull/31702/commits/abf8e13f5bd4c209d7176475edf4ff4f0efb95b3